templates/helpers.js

```diff
@@ -240,8 +241,7 @@ module.exports = function(docMap, options, getCurrent, helpers, OtherHandlebars)
                 return false;
             }
             var name = packageObject.name,
```

Comment:
@leoj3n do you know what `packageObject.name` is? Is it intended to be the parent package name? So in this case `canjs`. Or is it meant to be the name of the current package being processed?